### PR TITLE
Added Authentication for navbar

### PR DESCRIPTION
**Pull request by denimozh**
Added Authentication for navbar

### CODE REVIEW

File: src/app/api/orders/route.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                }
+            });
+            return new NextResponse(JSON.stringify(orders), {status:200});
+    
+        } catch (error) {
+            console.log(error);
+            return new NextResponse(JSON.stringify({message:"Something went wrong!"}), {status:500});
+        }
+    } else {
+        return new NextResponse(JSON.stringify({message:"Not Authenticated!"}), {status:401});
```
Comment (AliakseiDeboi) [inline src/app/api/orders/route.ts:30]: You can rewrite this code. Try catch block must be on top, then if without else (your code will execute the last statement only if the condition if false

File: src/app/api/orders/route.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+    
+}
+
+export const POST = () => {
+    return new NextResponse("Hello", {status:200});
```
Comment (AliakseiDeboi) [inline src/app/api/orders/route.ts:36]: Statuses and texts must be in separate enums

File: src/utils/auth.ts [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+                    email: token.email!
+                },
+            })
+            token.isAdmin = userInDb?.isAdmin!
+            return token;
+        }
+    }
+}
+
+export const getAuthSession = () => getServerSession(authOptions);
```
Comment (AliakseiDeboi) [inline src/utils/auth.ts:51]: ESLINT

**Review by AliakseiDeboi — CHANGES_REQUESTED** [review]

File: src/app/api/orders/route.ts (hunk 2 above)
Comment (denimozh) [reply src/app/api/orders/route.ts:36]: Will do this in lesson - I'm not too sure how

File: src/utils/auth.ts (hunk 3 above)
Comment (denimozh) [reply src/utils/auth.ts:51]: Okay will install

File: src/app/api/orders/route.ts (hunk 1 above)
Comment (denimozh) [reply src/app/api/orders/route.ts:30]: I will change

**Pull request merged**